great_expectations/render/renderer/checkpoint_new_notebook_renderer.py

```diff
@@ -132,7 +132,7 @@ def _add_sample_checkpoint_config(self) -> None:
                 sample_yaml_str,
                 lint=True,
             )
-        except:
+        except Exception:
```

Comment:
Aren't these the same thing?

Reply:
The liter doesn't like it. I think the bare except may also catch `KeyBoardInterrupt` exceptions which doesn't inherit from `Exception`
https://docs.python.org/3/library/exceptions.html#exception-hierarchy